index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <!-- Meta tags  -->
+   <meta charset="utf-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+   <!-- End Meta tags  -->
+
+   <!-- Title  -->
+   <title>Software Developer Portfolio | Nwachan Theophile</title>
+   <!-- End Title  -->
+
+   <!-- Styles  -->
+   <link rel="stylesheet" href="css/style.css">

```

Comment:
- [ ] This is the wrong path to your CSS file and as a result, your styles have not been loaded. Use the correct path (`href="style.css"`) and make sure your page looks like the design on Figma.
- Your page:
![Capture1](https://user-images.githubusercontent.com/85113567/153581975-60e0d457-3a0b-4aae-a67a-faa1595cc466.PNG)